includes/class-coblocks-block-assets.php

```diff
@@ -479,6 +490,21 @@ public function has_coblocks_block( WP_Post $post_object ) {
 
 	}
 
+	/**
+	 * Determine if the given post content contains any v1 Masonry block.
+	 *
+	 * @access public
+	 * @since  2.22.0
+	 *
+	 * @return boolean True when post content contains a v1 Masonry block.
+	 */
+	public function has_masonry_v1_block() {
+		$v1_regex = '/<!-- wp:coblocks\/gallery-masonry.*|\n*(coblocks-gallery--item).*|\n*<!-- \/wp:coblocks\/gallery-masonry -->/m';
+
+		preg_match_all( $v1_regex, get_the_content(), $matches );
+		return isset( $matches[0] ) && isset( $matches[0][2] ) && ! empty( $matches[0][2] );
```

Comment:
We don't need to store and perform conditions on any of the matches, we just want to know if there is a match. This can be simplified with:
```
return preg_match( $v1_regex, get_the_content() );
```